### PR TITLE
CAD-388 cluster3nodes with RTview

### PR DESCRIPTION
**Pull request by CodiePP**
* updates cardano-node and cardano-db-sync
* cluster3nodes can be run with RTview

### CODE REVIEW

File: cardano-rt-view/src/Cardano/Benchmarking/RTView/Acceptor.hs [hunk 1]
```diff
@@ -26,6 +27,7 @@ launchMetricsAcceptor
   -> Trace IO Text
   -> Switchboard Text
   -> IO ()
-launchMetricsAcceptor config accTr switchBoard = do
-  TraceAcceptor.plugin config accTr switchBoard >>= loadPlugin switchBoard
-  forever $ threadDelay 1000000
+launchMetricsAcceptor config accTr switchBoard =
+  withIOManager $ \iomgr -> do
```
Comment (CodiePP) [inline cardano-rt-view/src/Cardano/Benchmarking/RTView/Acceptor.hs]: @denisshevchenko : new requirement by updated iohk-monitoring fw

**Comment by deepfire** [comment]
This also disables the explorer.

File: benchmarks/cluster3nodes/configuration/log-config-rt-view-2.yaml [hunk 2]
```diff
@@ -98,7 +98,7 @@ traceForwardTo:
 
 NodeId:
 Protocol: RealPBFT
-GenesisFile: configuration/genesis/latest-genesis
+GenesisFile: some/path
```
Comment (deepfire) [inline benchmarks/cluster3nodes/configuration/log-config-rt-view-2.yaml:101]: Is this intended?

File: cardano-rt-view/src/Cardano/Benchmarking/RTView/Acceptor.hs (hunk 1 above)
Comment (deepfire) [reply cardano-rt-view/src/Cardano/Benchmarking/RTView/Acceptor.hs]: :100:

**Review by deepfire — APPROVED** [review]
LGTM, with just one question.

File: benchmarks/cluster3nodes/configuration/log-config-rt-view-2.yaml (hunk 2 above)
Comment (CodiePP) [reply benchmarks/cluster3nodes/configuration/log-config-rt-view-2.yaml:101]: yes, configuration is being overwritten by the start script with the absolute path to the genesis file. also a new requirement.

**Pull request merged**